editor/test/integration/reload_test.clj

```diff
@@ -805,7 +805,7 @@
     (let [[workspace project] (setup-scratch world)]
       (let [all-files (->>
                         (workspace/resolve-workspace-resource workspace "/")
-                        (tree-seq (fn [r] (and (not (resource/read-only? r)) (resource/children r))) resource/children)
+                        (tree-seq (fn [r] (and (resource/editable? r) (not (resource/read-only? r)) (resource/children r))) resource/children)
```

Comment:
This is the set of resources that are expected to produce `save-data`, and non-editable resources shouldn't do that.